frontend/src/components/single-blog/single.blog.jsx

```diff
@@ -1,58 +1,65 @@
-import React from "react";
+import { SectionHeader, BlogSections, AboutAuthor, Container } from "shared-components";
+import BlogsService from "pages/api/blogsService";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
 
-import { SectionHeader, BlogArticles, AboutDetails, Container } from "shared-components";
+let loaded = false;
+let currentBlogId = 0;
```

Comment:
Ove dve varijable bi trebalo premestiti u samu komponenti koristeci `useState`.
Boolean varijable bi trebalo da pocinju sa prefiksima `is`, `has`, `should`, `was` itd. 
`loaded` bi trebalo da se preimenuje u `hasLoaded` ili `isLoading`. Obicno se koristi `isLoading` koji je po default-u `true`, a u samoj komponenti, ako se ona i dalje ucitava (`isLoading` je `true`), vracas `null` umesto JSX-a.